### PR TITLE
Implement marker tri-selection and display angle calculation 

### LINKED ISSUE #161: Implement marker tri-selection

### LINKED ISSUE #162: Display angle calculation

### PR DESCRIPTION
**Pull request by javitrejo14**
Closes #161 and closes #162

### CODE REVIEW

**Review by PTLux1517 — APPROVED** [review]

File: src/SelectionInfoView.tsx [hunk 1]
```diff
@@ -1,34 +1,58 @@
 import * as React from "react";
 
-import { MarkerFileData } from "./DataTypes";
+import { MarkerFileData, MarkerFrame } from "./DataTypes";
+import { computeAngle} from "./Calculations";
```
Comment (PTLux1517) [inline src/SelectionInfoView.tsx:4]: It looks like the MarkerFrame import isn't needed any more, so line 3 can be reverted to how it was before

Comment (javitrejo14) [reply src/SelectionInfoView.tsx:4]: Oh yeah, fixed.

File: src/SelectionInfoView.tsx [hunk 2]
```diff
@@ -1,34 +1,58 @@
...
 }
 
-export default function SelectionInfoView(props: Props) {
-	if(props.selectedMarkers.length === 0) return <p>Nothing selected.</p>;
-	return <div>
-		{
-			props.selectedMarkers.map(idx => {
-				const info = props.markerData.markers[idx];
-				const frameInfo = props.markerData.frames[props.frame].positions[idx];
+	export default function SelectionInfoView(props: Props) {
```
Comment (vpzomtrrfrt) [inline src/SelectionInfoView.tsx:12]: indentation is wrong here

File: src/SelectionInfoView.tsx [hunk 3]
```diff
@@ -1,34 +1,58 @@
...
+									z: {frameInfo.z}
+								</>
+						}
+					</p>;
+				})
+			}
+		</>;
+	
+		const thetaLabel = props.markerData.markers[props.selectedMarkers[0]].label;
+		let angle: number|null = null;
```
Comment (vpzomtrrfrt) [inline src/SelectionInfoView.tsx:40]: could be moved closer to use and made const

File: src/SelectionInfoView.tsx [hunk 4]
```diff
@@ -1,34 +1,58 @@
...
+		let angleOutput;
+	
+		if (props.selectedMarkers.length >= 3) {
+			const first3Pos = props.selectedMarkers.slice(0,3).map(idx => props.markerData.frames[props.frame].positions[idx]);
+			angle = computeAngle(first3Pos);
+			if (angle!==null) {
+				angle = Math.round(10 * angle) / 10; //round to nearest tenth
+				angleOutput = <p>{thetaLabel} Angle: {angle} deg</p>;
```
Comment (vpzomtrrfrt) [inline src/SelectionInfoView.tsx:48]: could use degree symbol?

File: src/SelectionInfoView.tsx [hunk 5]
```diff
@@ -1,34 +1,58 @@
 import * as React from "react";
 
 import { MarkerFileData } from "./DataTypes";
+import { computeAngle} from "./Calculations";
```
Comment (vpzomtrrfrt) [inline src/SelectionInfoView.tsx:4]: add another space

File: src/SelectionInfoView.tsx (hunk 2 above)
Comment (vpzomtrrfrt) [reply src/SelectionInfoView.tsx:13]: still relevant, all these lines are too far

Comment (javitrejo14) [reply src/SelectionInfoView.tsx]: I see what you mean now. Fixed.

**Comment by javitrejo14** [comment]
Had a meeting outside of group meetups with Cory on this to get me up to speed with new code implementations as I had to use them. Spent about 2-3 hours on Sunday trying to figure out how to go about accomplishing this task, before meeting with Cory. Then another 3 hours implementing the angle calculation from the 3 markers selected.

**Pull request merged**